app/policies/collection_policy.rb

```diff
@@ -1,6 +1,6 @@
+# frozen_string_literal: true
```

Comment:
As mentioned before, remove this and ignore frozen_string_literal warnings from rubocop. Also, edit rubocop.yml to ignore these warnings.